palanaeum/templates/palanaeum/pagination_nav.html

```diff
@@ -5,10 +5,12 @@
     {% for page_num in page_numbers_to_show %}
         {% if page_num == page.number %}
             <a href="#" class="button1-accent w3-bar-item">{{ page_num }}</a>
-        {% elif page_num == -1 %}
-            <a href="#" class="button1 w3-bar-item">&hellip;</a>
+        {% elif page_num == 0 %}
+            <div onclick="show_hidden_pages(this)" class="button1 w3-bar-item">&hellip;</div>
+        {% elif page_num < 0 %}
+            <a href="{{ url }}?page={{ page_num|stringformat:'d'|slice:'1:' }}&{{ page_params }}" class="button1 w3-bar-item hidden-pg-num">{{ page_num|stringformat:'d'|slice:'1:' }}</a>
```

Comment:
I don't understand why are you `|slice`ing  the number after you format... wouldn't just {{ page_num }} work fine?

Reply:
I am using negative numbers to signal to the HTML template that it should start hidden. But we don't want the numbers to be negative in the link or text. I didn't see a multiplication/absolute value filter in Django and didn't know if you were okay with me adding a new filter, so I did this to get rid of the '-' in front. Should I create a filter instead?